wrappers/python/nntile/model/llama.py

```diff
@@ -88,6 +89,23 @@ def from_torch(torch_llama,
                                     config.hidden_size_tile,
                                     config.hidden_size_tile,
                                     next_tag)
+        elif config.dtype == "bf16":
+            embed_layer, next_tag = Embedding.generate_simple(
+                                    x_value, Tensor_bf16, 0,
```

Comment:
make all `if` statements only set variable `tensor_type` to one of `Tensor_fp32`, ... Create embedding layer after the new shortened `if` clause.

Reply:
Alternatively, a `dict` to map string representation to a proper Tensor type can be used